Wrappers/Python/cil/processors/Padder.py

```diff
@@ -28,258 +29,440 @@ class Padder(DataProcessor):
     '''
 
     @staticmethod
-    def constant(pad_width=None, constant_values=0):
-        '''
+    def constant(pad_width=None, constant_values=0.0):
+        """
         Padder processor wrapping numpy.pad with mode `constant` 
-        Pads with a constant value.
+        Pads the data with a constant value border.
+        Pads in all *spatial* dimensions unless a dictionary is passed to either `pad_width` or `constant_values`
```

Comment:
From this I understand that I could force padder to pad in any (non spatial) dimension if the dictionary is passed. But I understood from you that angles will never be padded?

Reply:
Changed to take angle